main.py

```diff
@@ -338,10 +376,13 @@ def update_game(self):
                 if abs(self.player.x - enemy.x) < 6:
                     if abs(self.player.y - enemy.y) < 6:
                         if isinstance(enemy, Fire):
-                            self.player.alive = False
-                            self.play_death_sound()
+                            if not self.player.is_invicible:
+                                self.player.alive = False
+                                self.play_death_sound()
                         else:
                             self.winner = True
+                            # ... To disable scrolling calculations
+                            self.player.alive = False
```

Comment:
Ah, another great fix included! I just noticed yesterday, but haven't fixed it yet! :tada:

Reply:
Yes, the view was still scrolling after winning 😆